### ISSUE TITLE
[BUG]presets="best_quality", the TimeSeriesPredictor won't train AutogluonTabular Model

### ISSUE THREAD

**Issue by xiaochenbin9527** [autogluon/autogluon#2997]
- [x] I have checked that this bug exists on the latest stable version of AutoGluon
- [x] and/or I have checked that this bug exists on the latest mainline of AutoGluon via source installation

**Describe the bug**
I'm trying to follow the "Forecating Time Series - Quick Start" and I found that when I used the  presets="best_quality", the TimeSeriesPredictor won't train AutogluonTabular Model. As I set the presets="high_quality" or presets="medium_quality", the TimeSeriesPredictor will train the AutogluonTabular Model.

**Expected behavior**
I think it is necessary to train AutogluonTabular Model when set the presets="best_quality" to find a best result, sometimes AutogluonTabular Model will get a better result.

**To Reproduce**
```
import pandas as pd
from autogluon.timeseries import TimeSeriesDataFrame, TimeSeriesPredictor
df = pd.read_csv("https://autogluon.s3.amazonaws.com/datasets/timeseries/m4_hourly_subset/train.csv")
train_data = TimeSeriesDataFrame.from_data_frame(
    df,
    id_column="item_id",
    timestamp_column="timestamp"
)
predictor = TimeSeriesPredictor(
    prediction_length=48,
    path="autogluon-m4-hourly",
    target="target",
    eval_metric="sMAPE",
)

predictor.fit(
    train_data,
    presets="best_quality"
)

```
**Screenshots**
![截图 2023-03-05 03-00-49](https://user-images.githubusercontent.com/39798501/222924480-1da3ac00-26d9-4813-acfe-6e2ca816a3dd.png)
![截图 2023-03-05 03-00-23](https://user-images.githubusercontent.com/39798501/222924497-bef8334d-1591-4e3c-b170-89fee81b593d.png)

**Installed Versions**
0.7.0

```python

INSTALLED VERSIONS
------------------
date                   : 2023-03-05
time                   : 03:12:48.046495
python                 : 3.8.16.final.0
OS                     : Linux
OS-release             : 5.19.0-35-generic
Version                : #36~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Feb 17 15:17:25 UTC 2
machine                : x86_64
processor              : x86_64
num_cores              : 20
cpu_ram_mb             : 31913
cuda version           : 11.510.108.03
num_gpus               : 1
gpu_ram_mb             : [8034]
avail_disk_size_mb     : 367581

accelerate             : 0.16.0
autogluon.common       : 0.7.0
autogluon.core         : 0.7.0
autogluon.features     : 0.7.0
autogluon.multimodal   : 0.7.0
autogluon.tabular      : 0.7.0
autogluon.timeseries   : 0.7.0
boto3                  : 1.26.84
catboost               : 1.1.1
defusedxml             : 0.7.1
evaluate               : 0.3.0
fairscale              : 0.4.13
fastai                 : 2.7.11
gluonts                : 0.12.3
hyperopt               : 0.2.7
jinja2                 : 3.1.2
joblib                 : 1.2.0
jsonschema             : 4.17.3
lightgbm               : 3.3.5
matplotlib             : 3.7.1
networkx               : 2.8.8
nlpaug                 : 1.1.11
nltk                   : 3.8.1
nptyping               : 2.4.1
numpy                  : 1.23.5
omegaconf              : 2.2.3
openmim                : None
pandas                 : 1.5.3
PIL                    : 9.4.0
psutil                 : 5.9.0
pytesseract            : 0.3.10
pytorch-metric-learning: None
pytorch_lightning      : 1.9.4
ray                    : 2.2.0
requests               : 2.28.2
scipy                  : 1.10.1
sentencepiece          : 0.1.97
seqeval                : None
setuptools             : 65.6.3
skimage                : 0.19.3
sklearn                : 1.2.1
statsforecast          : 1.4.0
statsmodels            : 0.13.5
tensorboard            : 2.12.0
text-unidecode         : None
timm                   : 0.6.12
torch                  : 1.13.1+cu117
torchmetrics           : 0.8.2
torchvision            : 0.14.1+cu117
tqdm                   : 4.65.0
transformers           : 4.26.1
ujson                  : 5.7.0
xgboost                : 1.7.4

```

**Comment by shchur** [comment]
Thank you, this is a good point. As a temporary workaround you can manually copy the `hyperparameters` from [`presets.py`](https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/models/presets.py#L144) as follows

```python
import autogluon.core as ag

context_length = max(prediction_length * 2, 10)

predictor.fit(
    train_data,
    hyperparameters={
        "Naive": {},
        "SeasonalNaive": {},
        "ARIMA": {},
        "ETS": {},
        "AutoETS": {},
        "AutoARIMA": {},
        "DynamicOptimizedTheta": {},
        "Theta": {
            "deseasonalize": ag.Categorical(True, False),
        },
        "DeepAR": {
            "context_length": context_length,
            "num_layers": ag.Int(1, 3, default=2),
            "hidden_size": ag.Int(40, 80, default=40),
        },
        "SimpleFeedForward": {
            "context_length": context_length,
            "hidden_dimensions": ag.Categorical([40], [40, 40], [120]),
        },
        "TemporalFusionTransformer": {},
        "AutoGluonTabular": {},
    },
    hyperparameter_tune_kwargs={
        "scheduler": "local",
        "searcher": "auto",
        "num_trials": 10,
    },
)
```

**Comment by shchur** [comment]
Fixed by #2998
